exercise01.cu

```diff
@@ -6,14 +6,15 @@
 //The number of character in the encrypted text
 #define N 1024
 
-//Decryption values
-#define A 111
-#define B 27
-#define M 127
-
 void checkCUDAError(const char*);
 void read_encrypted_file(int*);
 
+//Decryption values
+#define A 15
+#define B 27
+#define M 128
+#define A_MMI_M 111
+
```

Comment:
I moved the order of the  macros in the file to match those from the solution (for better diff'ing)

Reply:
Excellent